### PR TITLE
Adding Endpoints

### PR DESCRIPTION
**Pull request by ms-alguerra**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@t-alguer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (t-alguer). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/Commands.TrafficManager2.Test.csproj [hunk 1]
```diff
@@ -46,7 +46,7 @@
     </Reference>
     <Reference Include="Microsoft.Azure.Common.Authentication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.27-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.26-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
```
Comment (markcowl) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/Commands.TrafficManager2.Test.csproj]: Do not downgrade this package

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2/Commands.TrafficManager2.csproj [hunk 2]
```diff
@@ -53,10 +53,10 @@
     </Reference>
     <Reference Include="Microsoft.Azure.Common.Authentication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.27-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.26-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
```
Comment (markcowl) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2/Commands.TrafficManager2.csproj]: Do not downgrade this package

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2/packages.config [hunk 3]
```diff
@@ -2,10 +2,10 @@
 <packages>
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Common.Authentication" version="1.0.27-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.0.26-preview" targetFramework="net45" />
```
Comment (markcowl) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2/packages.config]: Do not downgrade this package

**Comment by markcowl** [comment]
@t-alguer: please address review comments

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@t-alguer tests fail, please address review comments and fix the ci build break.

**Comment by t-alguer** [comment]
Comments addressed. I am waiting for SDK review to get the build passing. But we don't want to publish any just yet. We will let you know on Monday.

Thanks, Alberto

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by markcowl** [comment]
@azuresdkci reetest this please

**Comment by markcowl** [comment]
@t-alguer There is a failure in traffic manager tests

**Comment by markcowl** [comment]
@t-alguer closing this in favor of Vlad's PR

**Pull request closed without merging**